firmware/fsm_msg_stellar.h

```diff
@@ -96,7 +96,7 @@ void fsm_msgStellarSignTx(StellarSignTx *msg)
 
 void fsm_msgStellarCreateAccountOp(StellarCreateAccountOp *msg)
 {
-    stellar_confirmCreateAccountOp(msg);
+    if (!stellar_confirmCreateAccountOp(msg)) return;
 
     if (stellar_allOperationsConfirmed()) {
```

Comment:
Could you consolidate this `if` statement (which is repeated in all the `fsm_msg*Op`) into a `stellar_requestOp` or similar function?

Reply:
Will do!